.github/workflows/build_and_test.yml

```diff
@@ -47,12 +47,13 @@ jobs:
                               ${{ matrix.use_s3_obj_store }} \
                               -DUSE_OPENTRACING=ON \
                               -DOMIT_TEST_OUTPUT=OFF\" "\
-              && script -q -e -c "make test"
+              && script -q -e -c "make test -DKEEP_APOLLO_LOGS=TRUE"
```

Comment:
The test fails because `make` does not accept variables in this way, error message:
```
2020-09-24T16:07:01.4392760Z Build finished. The binaries are in /home/runner/work/concord-bft/concord-bft/build
2020-09-24T16:07:01.4622684Z make: invalid option -- 'D'
2020-09-24T16:07:01.4623320Z make: invalid option -- 'K'
2020-09-24T16:07:01.4623799Z make: invalid option -- 'E'
2020-09-24T16:07:01.4624271Z make: invalid option -- 'E'
2020-09-24T16:07:01.4624745Z make: invalid option -- 'P'
2020-09-24T16:07:01.4625213Z make: invalid option -- '_'
2020-09-24T16:07:01.4625682Z make: invalid option -- 'A'
2020-09-24T16:07:01.4626154Z make: invalid option -- 'P'
```